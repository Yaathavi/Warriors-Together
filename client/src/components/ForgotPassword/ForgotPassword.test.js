/**
 * @jest-environment jsdom
 */

import React from "react";
import renderer from "react-test-renderer";
import ForgotPassword from "./index";

it("Renders the correct component", () => {
  const component = renderer.create(
    <ForgotPassword />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
