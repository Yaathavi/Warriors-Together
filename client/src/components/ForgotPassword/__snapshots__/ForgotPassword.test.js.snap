// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the correct component 1`] = `
<div
  className="flex flex-col min-h-screen overflow-hidden bg-gradient-to-b from-gray-50 via-gray-100 to-gray-200"
>
  <div>
    <nav
      class="w-full z-30 top-0 text-blue-1000 py-1 lg:py-6"
    >
      <div
        class="w-full container mx-auto flex flex-wrap items-center justify-between mt-0 px-2 py-2 lg:py-6"
      >
        <div
          class="pl-4 flex items-center"
        >
          <a
            class="hover:no-underline font-bold text-2xl lg:text-4xl flex flex-row items-center"
            href="/"
          >
            <img
              className="w-20 mr-4"
              src="./logo.png"
            />
            <p
              className="text-5xl text-blue-1000"
            >
               WarriorsTogether 
            </p>
          </a>
        </div>
        <div
          class="w-full flex-grow lg:flex lg:items-center lg:w-auto hidden mt-2 lg:mt-0 text-black p-4 lg:p-0 z-20 justify-end"
          id="nav-content"
        >
          <button
            class="mr-3"
          >
            <a
              class="inline-block py-2 px-4"
              href="/"
            >
              <p
                className="text-lg text-blue-1000 border border-solid border-blue-1000 rounded-md px-4 py-2 font-bold"
              >
                Sign Up
              </p>
            </a>
          </button>
        </div>
      </div>
    </nav>
  </div>
  <div
    className="flex flex-col items-center justify-center ml-4"
  >
    <div
      class="w-full max-w-lg bg-gray-400 p-12 shadow-2xl"
    >
      <p
        className="text-center text-2xl font-bold mb-6 mt-4"
      >
         Forgot Password 
      </p>
      <div
        class="flex flex-wrap -mx-3 mb-4"
      >
        <label
          class="block uppercase tracking-wide  text-xs font-bold mb-2"
          for="grid-last-name"
        >
          Email
        </label>
        <input
          class="appearance-none block w-full bg-gray-200 text-gray-700 border rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white"
          id="email"
          name="email"
          onChange={[Function]}
          placeholder="email"
          type="email"
          value=""
        />
      </div>
      <button
        class="bg-blue-1000 hover:bg-gray-700 text-white font-bold py-2 px-4 w-full rounded border"
        id="submit-button"
        onClick={[Function]}
      >
        Send password reset email
      </button>
      <p
        className="text-center text-md text-red-700 italic mt-2 invisible"
      >
        No user with that email exists!
      </p>
    </div>
  </div>
</div>
`;
